specification/sql/resource-manager/readme.md

```diff
@@ -546,7 +546,7 @@ input-file:
 - ./Microsoft.Sql/preview/2021-05-01-preview/RestorableDroppedDatabases.json
 - ./Microsoft.Sql/preview/2021-05-01-preview/RestorableDroppedManagedDatabases.json
 - ./Microsoft.Sql/preview/2021-05-01-preview/ServerConnectionPolicies.json
-- ./Microsoft.Sql/preview/2021-11-01-preview/DistributedAvailabilityGroups.json
+- ./Microsoft.Sql/preview/2022-05-01-preview/DistributedAvailabilityGroups.json
```

Comment:
Can we put it at the bottom of this v5 SDK tag? 
actually, you don't need to add it in this PR. please refer step 9 of TSG

Reply:
sure, i'll revert it for now